content/docs/studio/user-guide/projects-and-experiments/live-metrics-and-plots.md

```diff
@@ -30,16 +23,25 @@ your password. That is, you can reset all your access credentials (your password
 and the access token) at once. This is handy if you suspect that your account
 security may have been compromised.
 
-## Configure your model training CI job
+## Configure your model training job
 
-You should define the following environment variables in your CI job:
+You should provide the following environment variables to your model training
+job:
 
 1.  `STUDIO_TOKEN`: The access token must be present in any request that sends
     data to the Iterative Studio ingestion endpoint. Requests with missing or
     incorrect access tokens are rejected with an appropriate HTTP error code and
     error message.
 
-    A secure way to provide the access token is to create a
+    If you are running the experiment locally, you can set this environment
+    variable when submitting the training job.
+
+    ```
+    $ STUDIO_TOKEN=**** dvc exp run
```

Comment:
I didn't include it intentionally, because it adds unnecessary complication here. It is possible to run a new experiment without supplied new params or values through Hydra. I can simply update the params in `params.yaml` and run the experiment.